src/react/features/filters/hooks/useTextFilter.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect } from 'react'
+import { useDispatch, useStore } from '../../../store'
+import { useCompare } from '../../../utils/useCompare'
+
+import { setFocus } from '../../navigation/actions'
+import { addDiory, updateDiory } from '../../room/actions'
+
+const SEARCH_TOOL_ID = 'SEARCH_TOOL_ID'
+
+export const useTurnOnTextFilter = () => {
```

Comment:
Is this used somewhere? I didn't find anything... Forgotten old one?

Reply:
you mean "useTurnOnTextFilter"; yes, I was moved to own file